app/templates/components/badge-bitbucket-pipelines.hbs

```diff
@@ -0,0 +1,6 @@
+<a href="https://bitbucket.org/delan/nonymous/addon/pipelines/home#!/results/branch/{{ branch }}/page/1">
```

Comment:
the /page/1 is mandatory (e.g. [this link](https://bitbucket.org/delan/iyypdgarimwgp/addon/pipelines/home#!/results/branch/branch%2Fname%2Fwith%2Fslashes) doesn’t work)

Reply:
This URL looks like it has your username in it? It should use `{{ repository }}`, right?

Reply:
good catch!

Reply:
fixed in 79f15b479fb03c48c3b07b393874cccd35689b07